### ISSUE TITLE
Incorrect heading structure inside sidebar

### ISSUE THREAD

**Issue by benoitchantre** [wprig/wprig#554]
## Issue Overview

`<h2>` should represent the main sections of a document.

When we look at the sidebar, each widget title use an `<h2>`. They do not represent main sections.
The sidebar should have a meaninful `<h2>` heading and widget titles should use `<h3>`

More info: https://rianrietveld.com/2014/11/21/html5-headings-in-wordpress-lets-fight/

Cc. @bamadesigner for feedback.
